flycheck.el

```diff
@@ -460,17 +460,32 @@ The value of this option is a function which is given the name or
 path of an executable and shall return the full path to the
 executable, or nil if the executable does not exit.
 
-The default is the standard `executable-find' function which
-searches `exec-path'.  You can customize this option to search
-for checkers in other environments such as bundle or NixOS
+The default is `flycheck--find-executable', which searches
```

Comment:
You haven't changed the default value though, only one of the options for customize.  I'm fine with changing the default since it should be backwards compatible.

If we change the default, we should bump the `package-version` to `32`.